### PR TITLE
[gas] add script to generate gas schedule update proposal

### PR DESCRIPTION
**Pull request by vgao1996**
This adds a script to generate the gas schedule update proposal.

To run the script:
```
cargo run -p aptos-gas -o <output path>
```

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4471)

### CODE REVIEW

**Comment by vgao1996** [comment]
Hmm... I probably messed up the commit history... Let me fix it

File: aptos-move/aptos-gas/src/gen.rs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    let mut pack = PackageBuilder::new("GasScheduleUpdate");
+
+    pack.add_source(
+        "update_gas_schedule.move",
+        &generate_script(&current_gas_schedule())?,
+    );
+    // TODO: use relative path here
+    pack.add_local_dep("AptosFramework", &aptos_framework_path().to_string_lossy());
+
+    pack.write_to_disk(args.output.as_deref().unwrap_or("./proposal"))?;
```
Comment (davidiw) [inline aptos-move/aptos-gas/src/gen.rs:122]: probably `proposal.move`?

**Review by davidiw — APPROVED** [review]
might recommend just shoving this into aptos-gas rather than making another crate.

File: aptos-move/aptos-gas/src/gen.rs (hunk 1 above)
Comment (vgao1996) [reply aptos-move/aptos-gas/src/gen.rs:122]: It's a move package.

**Comment by vgao1996** [comment]
@davidiw it's already in the captos-gas` crate. Notice that the new crate I added is a helper to generate Move packages, which I moved out of `e2e-move-tests` due to a cyclic dependency issue.

**Comment by davidiw** [comment]
I ... am confused :) let's get this in!

**Review by movekevin — APPROVED** [review]

**Review by sherry-x — APPROVED** [review]

**Pull request merged**